js/enquiry.js

```diff
@@ -43,7 +43,9 @@ $(document).ready(function() {
       },
       submitHandler: function(form) {
         if (!form.email.value.match(/^(.+)\@(\w+)\.(\w+)$/)) {
-          alert('Please input valid email.')
+          thisFormValidate.showErrors({
+            'email': 'Please enter a valid email address.'
+          })
```

Comment:
Checkout https://jqueryvalidation.org/jQuery.validator.addMethod ?

Add a `strictEmail` This is more jquery.validate way.